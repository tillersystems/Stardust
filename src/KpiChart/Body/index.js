import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

/**
 * KpiChart Body
 *
 * This component is in charge of displaying
 * the body of a kpi chart
 *
 * @param {string} children // children.
 * @param {string} className // className.
 *
 * @return {jsx}
 */
const KpiChartBody = ({ children, className }) => (
  <div className={className} data-testid="kpichart-body">
    {children}
  </div>
);

/**
 * PropTypes Validation
 */
const { node, string } = PropTypes;
KpiChartBody.propTypes = {
  children: node,
  className: string,
  // eslint-disable-next-line react/no-unused-prop-types
  height: string,
};

/**
 * Default props
 */
KpiChartBody.defaultProps = {
  children: null,
  className: '',
  height: '37.3rem',
};

export default styled(KpiChartBody)`
  height: ${({ height }) => (/^\d+(rem|px)$/.test(height) ? height : '37.3rem')};
`;
