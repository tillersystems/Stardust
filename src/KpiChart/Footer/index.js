import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const KpiChartFooter = ({ children, className }) => <div className={className}>{children}</div>;

/**
 * PropTypes Validation
 */
const { node, string } = PropTypes;
KpiChartFooter.propTypes = {
  children: node,
  className: string,
};

/**
 * Default props
 */
KpiChartFooter.defaultProps = {
  children: null,
  className: '',
};

export default styled(KpiChartFooter)`
  position: absolute;
  bottom: 1.5rem;
  left: 1rem;

  button {
    color: ${({ theme: { palette } }) => palette.primary.default};
  }
`;
