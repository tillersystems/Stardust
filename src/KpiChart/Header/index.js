import React from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const KpiChartHeader = ({ children, className }) => <div className={className}>{children}</div>;

/**
 * PropTypes Validation
 */
const { node, string } = PropTypes;
KpiChartHeader.propTypes = {
  children: node,
  className: string,
};

/**
 * Default props
 */
KpiChartHeader.defaultProps = {
  children: null,
  className: '',
};

export default styled(KpiChartHeader)`
  display: flex;
  justify-content: space-between;
  align-items: center;
`;
