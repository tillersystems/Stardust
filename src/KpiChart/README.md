# Kpi Chart

### Usage

```jsx
import { KpiChart } from '@tillersystems/stardust';
```

<!-- STORY -->

### Properties

| Name          | Required |   Type   | DefaultValue | Description                                                                          |
| ------------- | :------: | :------: | :----------: | ------------------------------------------------------------------------------------ |
| `children`    |    \*    | `string` |              | Anything that can be rendered: numbers, strings, elements or an array (or fragment). |
| `className`   |    -     | `string` |     `''`     | ClassName needed by styled components                                                |
| `isCompacted` |    \*    | `string` |              | If true, narrows the padding of the component and the font-size of the title         |

### Example

```jsx
import { KpiChart } from '@tillersystems/stardust';

render() {
  return <KpiChart isCompacted>
           <KpiChart.Header>
             <KpiChart.Title isCompacted>Title</KpiChart.Title>
           </KpiChart.Header>
           <KpiChart.Body height={`${height}px`}>...</KpiChart.Body>
           <KpiChart.Footer>
             ...
           </KpiChart.Footer>
         </KpiChart>
}
```
