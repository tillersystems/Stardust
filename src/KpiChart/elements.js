import styled from 'styled-components';

export const Title = styled.div`
  font-weight: 600;
  font-size: ${({ isCompacted, theme: { fonts } }) =>
    isCompacted ? fonts.size.medium : fonts.size.big};

  padding-bottom: ${({ isCompacted }) => (isCompacted ? '1.2rem' : '1.5rem')};

  border-bottom: 3px solid ${({ theme: { palette } }) => palette.primary.default};
  display: flex;
  align-items: center;
`;
