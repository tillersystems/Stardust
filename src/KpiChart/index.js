import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

import { Button } from '..';
import { Body, Header, Footer, Title } from './elements';

/**
 * Kpi Chart
 *
 * This component is in charge of displaying
 * a Kpi Chart
 *
 * @param {string} className // Class needed by styled component.
 *
 * @return {jsx}
 */

class KpiChart extends PureComponent {
  /** Prop types. */
  static propTypes = {
    className: PropTypes.string,
    title: PropTypes.string.isRequired,
    label: PropTypes.string.isRequired,
    render: PropTypes.func,
  };

  /** Default props. */
  static defaultProps = {
    className: '',
    render: () => {},
  };

  /** Internal state. */
  // state = {
  //   settings: false,
  // };

  /**
   * Handles click event on settings.
   */
  // handleClick = () => {
  //   const { settings } = this.state;
  //   this.setState({ ...this.state, settings: !settings });
  // };

  render() {
    const { className, title, label, render } = this.props;
    return (
      <div className={className}>
        <Header>
          <Title>{title}</Title>
          {/* <Button small ghost onClick={() => this.handleClick}>
            <Icon name="cog" color={Theme.palette.anthracite} width="1.6rem" height="1.6rem" />
          </Button> */}
        </Header>
        <Body>{render()}</Body>
        <Footer>
          <Button ghost>{label}</Button>
        </Footer>
      </div>
    );
  }
}

export default styled(KpiChart)`
  position: relative;

  display: flex;
  flex-direction: column;

  padding: 2.4rem 2rem;

  height: 100%;

  border-radius: ${({ theme: { dimensions } }) => dimensions.radius};

  border: 1px solid ${({ theme: { palette } }) => palette.gray};

  background: ${({ theme: { palette } }) => palette.white};

  font-size: 1.2rem;
`;
