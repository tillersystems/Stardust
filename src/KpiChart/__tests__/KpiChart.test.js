import React from 'react';

import KpiChart from '..';

describe('<KpiChart />', () => {
  test('should render without a problem', () => {
    const Header = 'My title';
    const Body = 'body';
    const Footer = 'footer';
    const { getByText } = render(
      <KpiChart>
        <KpiChart.Header>{Header}</KpiChart.Header>
        <KpiChart.Body>{Body}</KpiChart.Body>
        <KpiChart.Footer>{Footer}</KpiChart.Footer>
      </KpiChart>,
    );
    const HeaderNode = getByText(Header);
    const BodyNode = getByText(Body);
    const FooterNode = getByText(Footer);

    expect(HeaderNode).toBeInTheDocument();
    expect(BodyNode).toBeInTheDocument();
    expect(FooterNode).toBeInTheDocument();
  });

  test('should render kpi chart with a title without a problem', () => {
    const Title = 'My title';
    const { getByText } = render(
      <KpiChart>
        <KpiChart.Header>
          <KpiChart.Title>{Title}</KpiChart.Title>
        </KpiChart.Header>
        <KpiChart.Body>Body</KpiChart.Body>
        <KpiChart.Footer>Footer</KpiChart.Footer>
      </KpiChart>,
    );
    const TitleNode = getByText(Title);
    expect(TitleNode).toBeInTheDocument();
  });
});
