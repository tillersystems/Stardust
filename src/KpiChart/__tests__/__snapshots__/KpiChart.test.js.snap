// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<KpiChart /> should render with render props 1`] = `
.c5 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-left: 0;
  margin-bottom: 0;
  padding: 0.7rem 1.4rem;
  border-radius: 0.4rem;
  font-size: calc(1.2rem + 0.2rem);
  line-height: calc(1.2rem + 0.2rem);
  color: hsl(0,100%,100%);
  border: 1px solid;
  -webkit-transition: background 200ms ease;
  transition: background 200ms ease;
  cursor: pointer;
  outline: none;
  color: #dbdfe6;
  background: hsl(0,100%,100%);
  border: 1px solid hsl(218,18%,91%);
  color: hsl(213,17%,20%);
  background: none;
  border: none;
}

.c5:hover:not([disabled]) {
  background: #dbdfe6;
  border: 1px solid #dbdfe6;
  color: hsl(0,100%,100%)   color:current;
  background: none;
  border: none;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c2 {
  font-weight: 600;
  font-size: 1.6rem;
  padding-bottom: 0.5rem;
  border-bottom: 1px dotted hsl(213,17%,20%);
}

.c3 {
  height: 37.3rem;
}

.c4 {
  position: absolute;
  bottom: 1.5rem;
  left: 1rem;
}

.c4 button {
  color: hsl(200,74%,46%);
}

.c0 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 2.4rem 2rem;
  height: 100%;
  border-radius: 0.4rem;
  border: 1px solid hsl(218,18%,91%);
  background: hsl(0,100%,100%);
  font-size: 1.2rem;
}

<ThemeProvider
  theme={
    Object {
      "breakpoints": Object {
        "lg": 992,
        "md": 768,
        "sm": 576,
        "xl": 1200,
        "xs": 0,
      },
      "dimensions": Object {
        "big": "3.1rem",
        "bigInt": 3.1,
        "huge": "4.5rem",
        "hugeInt": 4.5,
        "medium": "1.5rem",
        "mediumInt": 1.5,
        "radius": "0.4rem",
        "radiusInt": 0.4,
        "small": "0.7rem",
        "smallInt": 0.7,
        "tiny": "0.3rem",
        "tinyInt": 0.3,
      },
      "fonts": Object {
        "family": "\\"Open Sans\\", Helvetica, Arial, sans-serif",
        "size": Object {
          "big": "3.2rem",
          "bigInt": 3.2,
          "huge": "6.4rem",
          "hugeInt": 6.4,
          "medium": "1.2rem",
          "mediumInt": 1.2,
          "small": "0.7rem",
          "smallInt": 0.7,
          "tiny": "0.3rem",
          "tinyInt": 0.3,
        },
        "spacing": Object {
          "big": "1.5px",
          "medium": "1px",
          "small": "0.5px",
        },
        "weight": Object {
          "normal": 400,
          "thick": 600,
          "thin": 200,
        },
      },
      "palette": Object {
        "alizarin": "hsl(6, 78%, 57%)",
        "anthracite": "hsl(213, 17%, 20%)",
        "black": "hsl(0, 0%, 0%)",
        "blackLight": "hsl(213, 17%, 20%)",
        "blue": "hsl(200, 74%, 46%)",
        "bodyBackground": "hsl(218, 29%, 95%)",
        "carrot": "hsl(28, 80%, 52%)",
        "clay": "hsl(215, 17%, 26%)",
        "darkBlue": "#1c88bf",
        "darkGray": "#dbdfe6",
        "darkGreen": "#7fbf3b",
        "darkRed": "#ea6151",
        "emerald": "hsl(145, 63%, 49%)",
        "fadeBlue": "hsl(210, 50%, 98%)",
        "failure": Object {
          "back": "hsl(6, 79%, 65%)",
          "darkBack": "#ea6151",
          "darkFore": "#f7f7f7",
          "fore": "hsl(0, 100%, 100%)",
          "lightBack": "#f1978d",
          "lightFore": "#fff",
        },
        "googleBrandRed": "#ea4335",
        "gray": "hsl(218, 18%, 91%)",
        "green": "hsl(89, 53%, 52%)",
        "grey": "hsl(206, 16%, 60%)",
        "greygoose": "hsl(216, 29%, 97%)",
        "lightBlue": "#3baae2",
        "lightGray": "#fff",
        "lightGreen": "#a0d16b",
        "lightRed": "#f1978d",
        "marble": "hsl(206, 23%, 69%)",
        "orange": "hsl(34, 91%, 65%)",
        "primary": Object {
          "back": "hsl(200, 74%, 46%)",
          "darkBack": "#1c88bf",
          "darkFore": "#f7f7f7",
          "fore": "hsl(0, 100%, 100%)",
          "lightBack": "#3baae2",
          "lightFore": "#fff",
        },
        "red": "hsl(6, 79%, 65%)",
        "river": "hsl(204, 70%, 53%)",
        "sand": "hsl(214, 8%, 84%)",
        "secondary": Object {
          "back": "hsl(0, 100%, 100%)",
          "darkBack": "#f7f7f7",
          "darkFore": "#8093a3",
          "fore": "hsl(206, 16%, 60%)",
          "lightBack": "#fff",
          "lightFore": "#a6b4be",
        },
        "success": Object {
          "back": "hsl(89, 53%, 52%)",
          "darkBack": "#7fbf3b",
          "darkFore": "#f7f7f7",
          "fore": "hsl(0, 100%, 100%)",
          "lightBack": "#a0d16b",
          "lightFore": "#fff",
        },
        "turquoise": "hsl(168, 76%, 42%)",
        "white": "hsl(0, 100%, 100%)",
        "wisteria": "hsl(282, 44%, 47%)",
      },
    }
  }
>
  <Styled(KpiChart)
    label="label"
    render={[Function]}
    title="title"
  >
    <KpiChart
      className="c0"
      label="label"
      render={[Function]}
      title="title"
    >
      <div
        className="c0"
      >
        <styled.div>
          <div
            className="c1"
          >
            <styled.div>
              <div
                className="c2"
              >
                title
              </div>
            </styled.div>
          </div>
        </styled.div>
        <styled.div>
          <div
            className="c3"
          >
            <p>
              Render Props
            </p>
          </div>
        </styled.div>
        <styled.div>
          <div
            className="c4"
          >
            <Button
              ghost={true}
            >
              <Button
                big={false}
                className=""
                disabled={false}
                failure={false}
                fluid={false}
                ghost={true}
                inverted={false}
                isGoogle={false}
                light={false}
                onClick={[Function]}
                primary={false}
                rounded={false}
                secondary={false}
                small={false}
                success={false}
                tiny={false}
                type="button"
              >
                <styled.button
                  big={false}
                  className=""
                  disabled={false}
                  failure={false}
                  fluid={false}
                  ghost={true}
                  inverted={false}
                  isGoogle={false}
                  light={false}
                  onClick={[Function]}
                  primary={false}
                  rounded={false}
                  secondary={false}
                  small={false}
                  success={false}
                  tiny={false}
                  type="button"
                >
                  <button
                    className="c5"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    label
                  </button>
                </styled.button>
              </Button>
            </Button>
          </div>
        </styled.div>
      </div>
    </KpiChart>
  </Styled(KpiChart)>
</ThemeProvider>
`;

exports[`<KpiChart /> should render withouth a problem 1`] = `
.c5 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-left: 0;
  margin-bottom: 0;
  padding: 0.7rem 1.4rem;
  border-radius: 0.4rem;
  font-size: calc(1.2rem + 0.2rem);
  line-height: calc(1.2rem + 0.2rem);
  color: hsl(0,100%,100%);
  border: 1px solid;
  -webkit-transition: background 200ms ease;
  transition: background 200ms ease;
  cursor: pointer;
  outline: none;
  color: #dbdfe6;
  background: hsl(0,100%,100%);
  border: 1px solid hsl(218,18%,91%);
  color: hsl(213,17%,20%);
  background: none;
  border: none;
}

.c5:hover:not([disabled]) {
  background: #dbdfe6;
  border: 1px solid #dbdfe6;
  color: hsl(0,100%,100%)   color:current;
  background: none;
  border: none;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c2 {
  font-weight: 600;
  font-size: 1.6rem;
  padding-bottom: 0.5rem;
  border-bottom: 1px dotted hsl(213,17%,20%);
}

.c3 {
  height: 37.3rem;
}

.c4 {
  position: absolute;
  bottom: 1.5rem;
  left: 1rem;
}

.c4 button {
  color: hsl(200,74%,46%);
}

.c0 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 2.4rem 2rem;
  height: 100%;
  border-radius: 0.4rem;
  border: 1px solid hsl(218,18%,91%);
  background: hsl(0,100%,100%);
  font-size: 1.2rem;
}

<ThemeProvider
  theme={
    Object {
      "breakpoints": Object {
        "lg": 992,
        "md": 768,
        "sm": 576,
        "xl": 1200,
        "xs": 0,
      },
      "dimensions": Object {
        "big": "3.1rem",
        "bigInt": 3.1,
        "huge": "4.5rem",
        "hugeInt": 4.5,
        "medium": "1.5rem",
        "mediumInt": 1.5,
        "radius": "0.4rem",
        "radiusInt": 0.4,
        "small": "0.7rem",
        "smallInt": 0.7,
        "tiny": "0.3rem",
        "tinyInt": 0.3,
      },
      "fonts": Object {
        "family": "\\"Open Sans\\", Helvetica, Arial, sans-serif",
        "size": Object {
          "big": "3.2rem",
          "bigInt": 3.2,
          "huge": "6.4rem",
          "hugeInt": 6.4,
          "medium": "1.2rem",
          "mediumInt": 1.2,
          "small": "0.7rem",
          "smallInt": 0.7,
          "tiny": "0.3rem",
          "tinyInt": 0.3,
        },
        "spacing": Object {
          "big": "1.5px",
          "medium": "1px",
          "small": "0.5px",
        },
        "weight": Object {
          "normal": 400,
          "thick": 600,
          "thin": 200,
        },
      },
      "palette": Object {
        "alizarin": "hsl(6, 78%, 57%)",
        "anthracite": "hsl(213, 17%, 20%)",
        "black": "hsl(0, 0%, 0%)",
        "blackLight": "hsl(213, 17%, 20%)",
        "blue": "hsl(200, 74%, 46%)",
        "bodyBackground": "hsl(218, 29%, 95%)",
        "carrot": "hsl(28, 80%, 52%)",
        "clay": "hsl(215, 17%, 26%)",
        "darkBlue": "#1c88bf",
        "darkGray": "#dbdfe6",
        "darkGreen": "#7fbf3b",
        "darkRed": "#ea6151",
        "emerald": "hsl(145, 63%, 49%)",
        "fadeBlue": "hsl(210, 50%, 98%)",
        "failure": Object {
          "back": "hsl(6, 79%, 65%)",
          "darkBack": "#ea6151",
          "darkFore": "#f7f7f7",
          "fore": "hsl(0, 100%, 100%)",
          "lightBack": "#f1978d",
          "lightFore": "#fff",
        },
        "googleBrandRed": "#ea4335",
        "gray": "hsl(218, 18%, 91%)",
        "green": "hsl(89, 53%, 52%)",
        "grey": "hsl(206, 16%, 60%)",
        "greygoose": "hsl(216, 29%, 97%)",
        "lightBlue": "#3baae2",
        "lightGray": "#fff",
        "lightGreen": "#a0d16b",
        "lightRed": "#f1978d",
        "marble": "hsl(206, 23%, 69%)",
        "orange": "hsl(34, 91%, 65%)",
        "primary": Object {
          "back": "hsl(200, 74%, 46%)",
          "darkBack": "#1c88bf",
          "darkFore": "#f7f7f7",
          "fore": "hsl(0, 100%, 100%)",
          "lightBack": "#3baae2",
          "lightFore": "#fff",
        },
        "red": "hsl(6, 79%, 65%)",
        "river": "hsl(204, 70%, 53%)",
        "sand": "hsl(214, 8%, 84%)",
        "secondary": Object {
          "back": "hsl(0, 100%, 100%)",
          "darkBack": "#f7f7f7",
          "darkFore": "#8093a3",
          "fore": "hsl(206, 16%, 60%)",
          "lightBack": "#fff",
          "lightFore": "#a6b4be",
        },
        "success": Object {
          "back": "hsl(89, 53%, 52%)",
          "darkBack": "#7fbf3b",
          "darkFore": "#f7f7f7",
          "fore": "hsl(0, 100%, 100%)",
          "lightBack": "#a0d16b",
          "lightFore": "#fff",
        },
        "turquoise": "hsl(168, 76%, 42%)",
        "white": "hsl(0, 100%, 100%)",
        "wisteria": "hsl(282, 44%, 47%)",
      },
    }
  }
>
  <Styled(KpiChart)
    label="label"
    title="title"
  >
    <KpiChart
      className="c0"
      label="label"
      render={[Function]}
      title="title"
    >
      <div
        className="c0"
      >
        <styled.div>
          <div
            className="c1"
          >
            <styled.div>
              <div
                className="c2"
              >
                title
              </div>
            </styled.div>
          </div>
        </styled.div>
        <styled.div>
          <div
            className="c3"
          />
        </styled.div>
        <styled.div>
          <div
            className="c4"
          >
            <Button
              ghost={true}
            >
              <Button
                big={false}
                className=""
                disabled={false}
                failure={false}
                fluid={false}
                ghost={true}
                inverted={false}
                isGoogle={false}
                light={false}
                onClick={[Function]}
                primary={false}
                rounded={false}
                secondary={false}
                small={false}
                success={false}
                tiny={false}
                type="button"
              >
                <styled.button
                  big={false}
                  className=""
                  disabled={false}
                  failure={false}
                  fluid={false}
                  ghost={true}
                  inverted={false}
                  isGoogle={false}
                  light={false}
                  onClick={[Function]}
                  primary={false}
                  rounded={false}
                  secondary={false}
                  small={false}
                  success={false}
                  tiny={false}
                  type="button"
                >
                  <button
                    className="c5"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    label
                  </button>
                </styled.button>
              </Button>
            </Button>
          </div>
        </styled.div>
      </div>
    </KpiChart>
  </Styled(KpiChart)>
</ThemeProvider>
`;
